src/library/scala/Predef.scala

```diff
@@ -371,6 +371,11 @@ object Predef extends LowPriorityImplicits {
     def +(other: String): String = String.valueOf(self) + other
   }
 
+  implicit final class UnliftOps[A, B] private[Predef](private val f: A => Option[B]) extends AnyVal {
```

Comment:
This should be in the companion object of `Function1`, I believe. There is no need for it to be in `Predef`.
Concrete advantage: faster compilation for everyone by not polluting the implicit scope that applies to *all* types.

Reply:
Moved `UnliftOps` to `genprod.scala`